db/migrate/20240309153813_create_contacts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateContacts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :contacts do |t|
+      t.string :name
+      t.string :relationship, default: "friend"
+      t.string :email
+      t.string :phone_number
+      t.string :address
+      t.string :social_media_handle, default: "@ig_name"
```

Comment:
Which social media? Instagram, fb, twitter?

Reply:
For now it was just insta, and later on we wanted to add a socials table to the user

Reply:
Then let's rename it `instagram_handle` 👌

Reply:
As discused  with @emmvs  we are going to go with a specific column/table for each handle.